lib/phoenix/live_dashboard/components/table_component.ex

```diff
@@ -186,6 +195,21 @@ defmodule Phoenix.LiveDashboard.TableComponent do
         </div>
       </form>
 
+    <%= if @filter do %>
+      <form phx-change="select_filter" phx-target={@myself} class="form-inline">
+        <div class="form-row align-items-center">
+            <div class="col-auto">Filter</div>
+            <div class="col-auto">
+              <div class="input-group input-group-sm">
+                <select name="filter" class="custom-select" id="filter-select">
+                  <%= options_for_select(@filter, @table_params.filter) %>
```

Comment:
For me, it isn't very clear because the first option should be a list but the variable's name is singular. Then the docs says it is a string: https://github.com/phoenixframework/phoenix_live_dashboard/pull/411/files#diff-6cd40a459178fef0d56adbf0efbf1bb2fbe754773d31aacb416b39b1911d4a11R291

I expected the `@filter` variable to store the current filter value, but I see now that is table params. I guess if we change the name to `@filters`, set the type to list, and optionally use `[]` as the default value would be better :)

Reply:
Thanks, I changed it, so we now have @filter for the selected filter value, and @filter_list for the list of values.